scss/utilities/_flex.scss

```diff
@@ -0,0 +1,35 @@
+// Flex variation
+//
+// Custom styles for additional flex alignment options.
+
+@if $enable-flex and $enable-grid-classes {
+
+  // Flex column reordering
+
+  @each $breakpoint in map-keys($grid-breakpoints) {
```

Comment:
Let's merge the three loops/includes into one, because it compiles to fewer media queries and is easier to read.